build/zip-examples.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+/*!
+ * Script to create the built examples zip archive;
+ * requires the `zip` command to be present!
+ * Copyright 2020 The Bootstrap Authors
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+
+/* eslint-disable camelcase */
+
+'use strict'
+
+const path = require('path')
+const sh = require('shelljs')
+
+const { version, version_short } = require('../package.json')
+
+const folderName = `bootstrap-${version}-examples`
+
+sh.config.fatal = true
+
+// switch to the root dir
+sh.cd(path.join(__dirname, '..'))
+
+// remove any previously create folder with the same name
+sh.rm('-rf', folderName)
+sh.mkdir('-p', folderName)
```

Comment:
it's a bit weird to use `sh` here, you can use native Node.js methods to do that, it'll work everywhere